packages/react-components/react-combobox/stories/Dropdown/DropdownTruncation.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Dropdown, makeStyles, Option, shorthands, useId } from '@fluentui/react-components';
+import type { DropdownProps } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    // Stack the label above the field with a gap
+    display: 'grid',
+    gridTemplateRows: 'repeat(1fr)',
+    justifyItems: 'start',
+    ...shorthands.gap('2px'),
```

Comment:
nit: use token here, just so we're always demonstrating them.

Reply:
Or better, use the `Field` component so you don't need these styles at all. (Maybe eventually update other dropdown stories to use Field, separately).

Reply:
I'm going to do this for all the stories separately, just leaving this here for now for consistency